src/server/list_family.cc

```diff
@@ -565,6 +563,68 @@ void ListFamily::LLen(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ListFamily::LPos(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+  string_view elem = ArgS(args, 2);
+
+  int rank = 1;
+  int count = -1;
+  int max_len = 0;
+
+  for (size_t i = 3; i < args.size(); i++) {
+    ToUpper(&args[i]);
+    const auto& arg_v = ArgS(args, i);
+    if (arg_v == "RANK") {
+      if (!absl::SimpleAtoi(ArgS(args, (i + 1)), &rank)) {
+        return (*cntx)->SendError(OpStatus::WRONG_TYPE);
```

Comment:
please use explicit `kInvalidIntErr` error message. Also you can unite both checks 
`!absl::SimpleAtoi(..) || rank == 0)`